packages/app/src/runner/SpecRunnerOpenMode.vue

```diff
@@ -234,6 +237,14 @@ onBeforeUnmount(() => {
   cleanupRunner()
 })
 
+const specsListWidthPreferences = computed(() => {
```

Comment:
Do we need these computed now that `spsecListWidth` and `reporterWidth` and being seeded with the gql values?